### PR TITLE
Drag and drop improvements

### PR DESCRIPTION
**Pull request by DelazJ**
(cherry picked from commit 5ceedee5d37745ae2912efd54f6b0bb073c7b108)

To merge only after https://github.com/qgis/QGIS/pull/34656

### CODE REVIEW

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 1]
```diff
@@ -1989,36 +1995,37 @@ in figure_fields_form_.
...
+   in the current layer's form. As for the other items, select the relation label
+   to configure some properties:
+
+   * hide or show the relation label
    * show the link button
    * show the unlink button
 
 #. Apply the layer's properties dialog
-#. Use the |identify| tool on a feature, and it should display the new form.
+#. Open a feature attribute form (eg, using the |identify| :sup:`Identify features`
```
Comment (havatv) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]:
```suggestion
#. Open a feature attribute form (using the |identify| :sup:`Identify features`
```

**Comment by DelazJ** [comment]
really??? This is a backport of a PR that was online for at least a week

**Comment by havatv** [comment]
Must have missed in the first round, then. Still room for improvements... :-)

File: source/docs/user_manual/working_with_vector/vector_properties.rst (hunk 1 above)
Comment (DelazJ) [reply source/docs/user_manual/working_with_vector/vector_properties.rst]: This is is not the single way to open a form, hence an example

**Pull request merged**